lib/twitter-ads/audiences/tailored_audience.rb

```diff
@@ -86,6 +88,25 @@ def create(account, file_path, name, list_type)
         end
       end
 
+      # Creates a new tailored audience.
+      #
+      # @example
+      #   audience = TailoredAudience.create_instance(account, 'my list')
+      #
+      # @param account [Account] The account object instance.
+      # @param name [String] The tailored audience name.
+      #
+      # @since 4.0
+      #
+      # @return [TailoredAudience] The newly created tailored audience instance.
+      def create_instance(account, name)
```

Comment:
Given that v4 will no longer support TON based audiences, I'd suggest removing this method in favor of editing the existing `create` method instead

Reply:
@tushdante 

There is a concern. The signatures of these two methods are totally different.  

```
create(account, file_path, name, list_type)
create_instance(account, name)
```

If I made a method like `create(account, name)`, the backward compatibility would be broken. Is this OK with you? If so, I would simply drop the current `create` method and rename `create_instance` to `create`.

Reply:
Good point! I think it should be fine to just drop the current `create` method and rename the `create_instance` to `create`, given that the current `create` method will no longer be supported as of v4.

Reply:
@tushdante Thank you for your comment. I understand. I will modify the code as you mentioned.